.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,42 @@
+name: ubuntu
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ '2.3.x', '2.4.x', '2.5.x', '2.6.x' ]
+        test_tool: [ "rubygems", "bundler" ]
+        include:
+          - ruby: 2.6.x
+            test_tool: bundler
+            rgv: ..
+            bdv: master
+    steps:
+      - uses: actions/checkout@master
+      - run: git submodule update -i
+      - name: Fixed world writable dirs
+        run: |
+          chmod go-w $HOME
+          sudo chmod -R go-w /usr/share
+          sudo chmod -R go-w /opt
+      - name: Setup ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+      - name: Install Dependencies
+        run: util/ci.sh before_script
+        env:
+          TEST_TOOL: ${{ matrix.test_tool }}
+      - name: Run Lint
+        run: util/ci.sh rubocop
+        env:
+          TEST_TOOL: rubygems
```

Comment:
Haven't yet digged into the new Github CI system, but if I undestand correctly, "jobs" would be TravisCI "stages". In our previous configuration, we considered enough to limit the "lint" stage to just the latest ruby, but here we're running it against all matrix entries. So perhaps add a separate "job" the run rubocop against a single ruby?

Reply:
GitHub Actions don't have the `stage` feature of Travis CI yet. If we want to separate the lint stage from the multiple ruby versions, we can add yaml file for only lint stage.

Reply:
So you can't add another top level job? If that's the case, a separate file is fine, or if you believe running rubocop for every ruby version is not too much, then we can leave it as it is.